### PR TITLE
Update getting_started.ipynb and getting_started.md

### PR DESCRIPTION
**Pull request by chiamp**
Replaced jax.value_and_grad with jax.grad in section 7 since the outputted loss value wasn't being used.

### CODE REVIEW

**Review by cgarciae — APPROVED** [review]
Looks good!
